bundles/binding/org.openhab.binding.maxcul/src/main/java/org/openhab/binding/maxcul/internal/MaxCulMsgHandler.java

```diff
@@ -275,7 +275,7 @@ public void checkPendingAcks() {
                     logger.debug("Retransmitting packet " + qi.msg.msgCount + " attempt " + qi.retryCount);
                     sendMessage(qi.msg, qi);
                 } else {
-                    logger.error("Transmission of packet " + qi.msg.msgCount + " failed 3 times");
+                    logger.error("Transmission of packet " + qi.msg.msgCount + " failed 3 times, message was " + qi.msg.msgType + " to address " + qi.msg.dstAddrStr + " => " + qi.msg.rawMsg);
```

Comment:
Awesome.  I don't think it's going to fix my problem (refer https://community.openhab.org/t/maxcul-retry-logic-proposed-extension/9983/4), but it'll take me some way down the path of knowing what's going on.